Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.h

```diff
@@ -137,6 +137,8 @@ extern "C" AXUIElementRef NSAccessibilityCreateAXUIElementRef(id element);
 
 // When a plugin uses a WebKit control to act as a surrogate view (e.g. PDF use WebKit to create text fields).
 - (id)_associatedPluginParent;
+// For testing use only.
+- (void)accessibilityHitTestResolvingRemoteFrame:(NSPoint)point callback:(void(^)(NSString *))callback;
```

Comment:
We should prefix this with `_` to indicate it's "private".

Reply:
Good call!